app/components/post/post.js

```diff
@@ -411,6 +414,7 @@ class Post extends PureComponent {
                             toggleSelected={this.toggleSelected}
                             managedConfig={managedConfig}
                             isFlagged={isFlagged}
+                            isReplyPost={this.isReplyPost()}
```

Comment:
Can this be moved up above the return and be assigned to a variable? Seems to strange to have it executing here.